StripePaymentsUI/StripePaymentsUI/Source/Helpers/STPAPIClient+CustomerContext.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  STPAPIClient+CustomerContext.swift
+//  StripePaymentsUI
+//
+//  Created by David Estes on 3/13/23.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+
+extension STPAPIClient {
+
+    /// Update a customer with parameters
+    /// - seealso: https://stripe.com/docs/api#update_customer
+    func updateCustomer(
+        withParameters parameters: [String: Any],
+        using ephemeralKey: _stpspmsbeta_STPEphemeralKey,
+        completion: @escaping STPCustomerCompletionBlock
+    ) {
+        let endpoint = "\(APIEndpointCustomers)/\(ephemeralKey.customerID ?? "")"
```

Comment:
Seems like we should `guard let customerID = ephemeralKey.customerID else { completion(...) }` ?